mne_bids/write.py

```diff
@@ -278,8 +279,22 @@ def _participants_tsv(raw, subject_id, fname, overwrite=False,
                                   'list. Please set overwrite to '
                                   'True.' % subject_id)
         # otherwise add the new data
+        _old_keys = data.keys()
         data = _combine(orig_data, data, 'participant_id')
 
+        # get the keys that were new
+        for key in set(orig_data.keys()) - set(_old_keys):
+            # add them back per participant id
+            for p_id, val in zip(data['participant_id'], data[key]):
```

Comment:
Not that I can think of off the top of my head.... Do you think there will be?

Reply:
My understanding of your current implementation is that it could happen that the order of the _columns_ in the generated TSV might change, is that correct? If so, I don't think this is desirable

Reply:
Ah I see, okay that makes sense! Fixed it.

Reply:
Nice! Could you add a test for this too? :)